### PR TITLE
Added form for Canopy_Cover and tests

### PR DESCRIPTION
**Pull request by cthalmann**
fixes issue #104 

## Changes in this PR.

- [X] Added form for Canopy_Cover
- [X] Added form for CC_Cardinal
- [X] Added tests 

## Testing this PR.

1. docker-compose run web bash
2. cd streamwebs_frontend
3. python manage.py test streamwebs/tests/forms

### Expected Output.

```
Creating test database for alias 'default'...
...........
----------------------------------------------------------------------
Ran 11 tests in 0.027s

OK
Destroying test database for alias 'default'...

```

@osuosl/devs

### CODE REVIEW

**Comment by leian7** [comment]
Looks good. Can you also look into implementing what @elenatorrey 's investigating right now, testing to see whether required form fields are actually required?

**Comment by alxngyn** [comment]
+1

File: streamwebs_frontend/streamwebs/forms.py [hunk 1]
```diff
@@ -83,3 +84,27 @@ class Meta:
...
+        }
+
+
+class CC_Cardinal_Form(forms.ModelForm):
+    class Meta:
+        model = CC_Cardinal
+        fields = ('direction', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I',
+                  'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U',
+                  'V', 'W', 'X', 'num_shaded')
+        A = forms.BooleanField(initial=False, required=False)
```
Comment (athai) [inline streamwebs_frontend/streamwebs/forms.py]: Is this line necessary?

Comment (cthalmann) [inline streamwebs_frontend/streamwebs/forms.py]: ^You are correct that it isn't, removing now.

**Comment by Kennric** [comment]
+1

**Comment by athai** [comment]
+1

**Pull request merged**